src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -4120,8 +4074,7 @@ protected void warning(String message, Exception ex) {
     }
 
     private void showErrorMessage(String message, String title) {
-        // TODO 736 fix this to show GUI message
-        System.err.println(title+": "+message); // TODO 736 remove this
+        JOptionPane.showMessageDialog(null,  message, title, JOptionPane.INFORMATION_MESSAGE);
```

Comment:
Two comments about this line:

1. Is it guaranteed at this point that noGUI==false?   Do we know for certain that invoking JOptionPane won't throw a HeadlessException?

2. The name of the method is `showErrorMessage()`.  If it is always the case that this method is called with an ERROR, I suggest `JOptionPane.ERROR_MESSAGE` is a more appropriate constant for the last argument in the call to `JOptionPane.showMessageDialog()`.

Reply:
Yes. showErrorMessage is only called if nogui is false.  I checked that.  
I was going to add a test in there for nogui, but checked and everyone who calls it (only a couple of places) checks already.

Reply:
The showErrorMessage() is a private method to InternalController.  It is called in exactly 2 places in InternalController.java and in both places, "usingGUI" is checked.

Reply:
2. Agreed.  JOptionPane.ERROR_MESSAGE is more appropriate (and puts a red X in the dialog instead of the "i").